### DISCUSSION TITLE
NOT_WORK

### DISCUSSION THREAD

**Discussion by leeonn2** [tiglat/catalog_maker#9] [General]
Total Commander 10.00
WIN 10
Плагин создаёт файл с заголовками и выдаёт ошибку «Write error!»(((
Перепробовал все выложенные версии.

**Comment by tiglat** [comment]
Добрый день! Попробовал установить новый Total Commander. Сейчас последняя версия 10.5 RC. Проблемы не обнаружил. Предлагаю проверить новую версию. Отпишитесь, пожалуйста, по результату.

**Comment by leeonn2** [comment]
Версия Тотала 10.51 (32 бита), создаёт список только для файлов в одном каталоге, если в качестве объекта он получает каталог с файлами или файлы лежащие в разных каталогах, получаем ошибку записи(((

**Reply by tiglat** [reply]
> файлы лежащие в разных каталогах
Уточните, что имеется в виду? Несколько каталогов с файлами?

**Reply by leeonn2** [reply]
> > файлы лежащие в разных каталогах
> > Уточните, что имеется в виду? Несколько каталогов с файлами?

Ну как выделяю несколько каталогов, и логично что на выходе хочу видеть файл, с перечислением что там лежит внутри этих каталогов, ну или по Ctrl+B вываливаю содержимое всех директорий, выбираю ручками или фильром нужные файлы и получаю список))))

**Reply by tiglat** [reply]
Попробовал установить 32 разрядный TotalCommander на Win10. Проблема не воспроизводится. 

Пришлите, пожалуйста, скриншот окно "О программе", которое можно открыть открыв настройки Windows, а затем кликнув на пункте Система. 

А также хотелось бы взглянуть на список файлов, который не удается обработать. Список файлов можно получить, открыв консоль Windows Power shell, кликнув правой кнопкой на кнопку Пуск. Затем нужно перейти в нужный каталог командой cd, и выполнить команду dir -r

**Comment by leeonn2** [comment]
Вот собственно сведения о системе:
![image](https://user-images.githubusercontent.com/80505559/217364404-07316be7-29bb-4e27-9bfa-a1e92f0d3524.png)

Сведения о тотале:
![image](https://user-images.githubusercontent.com/80505559/217364709-266d255c-8155-4c0e-9d7a-880e15002234.png)

Окно с ошибкой, это как вывидите обычна ДВД папка с видео:
![image](https://user-images.githubusercontent.com/80505559/217364221-d8957ae8-8f7b-4602-90f6-03b1e6fa0a51.png)

Вот список файлов в ней:
```
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VIDEO_TS.BUP
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VIDEO_TS.IFO
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_0.BUP
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_0.IFO
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_0.VOB
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_1.VOB
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_2.VOB
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_3.VOB
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_4.VOB
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_5.VOB
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_ALL.VOB
```

**Reply by leeonn2** [reply]
Ну собственно, если речь не идёт о файлах в одном каталоге, Ваш плагин рушится((((
Если зайти в каталог получается список файлов:
```
File name                                                       Size            Date        Time      Attr
------------------------------------------------------------    -------------   ----------  --------  ----
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VIDEO_TS.BUP                  8,192   09.01.2022  12:04.20  -a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VIDEO_TS.IFO                  8,192   09.01.2022  12:04.20  -a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_0.BUP                 73,728   09.01.2022  12:03.26  -a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_0.IFO                 73,728   09.01.2022  12:03.26  -a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_0.VOB              1,267,712   09.01.2022  12:03.26  -a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_1.VOB          1,073,565,696   09.01.2022  12:03.25  -a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_2.VOB          1,073,565,696   09.01.2022  12:03.24  -a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_3.VOB          1,073,565,696   09.01.2022  12:03.26  -a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_4.VOB          1,073,565,696   09.01.2022  12:03.26  -a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_5.VOB            337,408,000   09.01.2022  12:03.25  -a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_ALL.VOB        4,631,670,784   09.01.2022  12:19.39  -a--

total files 11    total size 9,264,773,120
```

**Reply by leeonn2** [reply]
Но то же самое я могу получить через встоенную в тотал команду:
`cm_CopyFpFileDetailsToClip`


Плюс она сохранит мои настройки вывода информации в колонках размер и дата :

```
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VIDEO_TS.BUP	8.19 Кб	2022.01.09 12:04	-a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VIDEO_TS.IFO	8.19 Кб	2022.01.09 12:04	-a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_0.BUP	73.72 Кб	2022.01.09 12:03	-a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_0.IFO	73.72 Кб	2022.01.09 12:03	-a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_0.VOB	1.26 Мб	2022.01.09 12:03	-a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_1.VOB	1.07 Гб	2022.01.09 12:03	-a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_2.VOB	1.07 Гб	2022.01.09 12:03	-a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_3.VOB	1.07 Гб	2022.01.09 12:03	-a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_4.VOB	1.07 Гб	2022.01.09 12:03	-a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_5.VOB	337.40 Мб	2022.01.09 12:03	-a--
e:\FILMS\!!!CLASIC\Belaja ptitsa\VIDEO_TS\VTS_01_ALL.VOB	4.63 Гб	2022.01.09 12:19	-a--
```


Радость плагина для меня была в том, чтобы запустить в корне, ну к примеру, диска (или папки) Ваш плагин и получить список файлов, то же самое можно и Ctrl+B вывалить всё в корне и вышеописанным способом копирнуть инфу, но это будет долго и ручками, хотелось от Вашего плуга, это автоматом получить)))))

**Reply by tiglat** [reply]
Спасибо за информацию. Попробуем разобраться. Надеюсь вы обретете радость от использования плагина )
А пока, хочу предложить установить 64 разрядную версию TotalCommander. У вас ведь 64 разрядная винда. Проблемы могут быть из-за этого тоже теоретически. https://learn.microsoft.com/ru-ru/troubleshoot/windows-server/performance/compatibility-limitations-32-bit-programs-64-bit-system

**Reply by tiglat** [reply]
Добрый вечер. Сообщите, пожалуйста, попробовали установить 64 разрядную версию? Помогло?

Я со своей стороны попытался наиграть ваш сценарий, создав подобную директорию с похожими файлами. К сожалению, проблема не воспроизвелась. Плагин работает штатно. Прикладываю запись своих эксперементов.
[bandicam 2023-03-11 19-21-05-121.zip](https://github.com/tiglat/catalog_maker/files/10948936/bandicam.2023-03-11.19-21-05-121.zip)
